content/en/docs/started/support.md

```diff
@@ -4,224 +4,99 @@ description = "Where to get support for Kubeflow"
 
 * The cause falls within the technical framework that Kubeflow controls. For
   example, the Kubeflow community may not be able to help if the problem is 
   caused by a specific network configuration within your organization.
 * Community members can reproduce the problem.
-* The reporter of the problem can help with further diagnosis and 
-  troubleshooting.
+* The reporter can assist with troubleshooting.
 
 You can ask questions and make suggestions in the following places:
 
-* **Slack** for online chat and messaging. See details of Kubeflow's 
-  [Slack workspace and channels](/docs/about/community/#kubeflow-slack-channels).
+* **Slack** for online chat and messaging, see [Slack workspace and channels](/docs/about/community/#kubeflow-slack-channels).
+* **GitHub discussions** per repository, e.g. [here](https://github.com/kubeflow/manifests/discussions)
 * **Kubeflow discuss** for email-based group discussion. Join the
   [kubeflow-discuss](/docs/about/community/#kubeflow-mailing-list)
   group.
 * **Kubeflow documentation** for overviews and how-to guides. In particular,
   refer to the following documents when troubleshooting a problem:
-
   * [Kubeflow installation and setup](/docs/started/installing-kubeflow/)
   * [Kubeflow components](/docs/components/)
 
 * **Kubeflow issue trackers** for known issues, questions, and feature requests.
   Search the open issues to see if someone else has already logged the problem 
-  that you're encountering and learn about any workarounds to date. If no one
+  that you are encountering and learn about any workarounds. If no one
   has logged your problem, create a new issue to describe the problem.
 
-    Each Kubeflow application has its own issue tracker within the [Kubeflow
+    Each Kubeflow component has its own issue tracker within the [Kubeflow
     organization on GitHub](https://github.com/kubeflow). To get you started,
     here are the primary issue trackers:
 
-  * [Kubeflow core](https://github.com/kubeflow/kubeflow/issues)
-  * [kfctl command-line tool](https://github.com/kubeflow/kfctl/issues)
-  * [Kustomize manifests](https://github.com/kubeflow/manifests/issues)
   * [Kubeflow Pipelines](https://github.com/kubeflow/pipelines/issues)
   * [Katib AutoML](https://github.com/kubeflow/katib/issues)
-  * [Metadata](https://github.com/kubeflow/metadata/issues)
-  * [Fairing notebook SDK](https://github.com/kubeflow/fairing/issues)
-  * [Kubeflow Training (TFJob, PyTorchJob, MXJob, XGBoostJob)](https://github.com/kubeflow/training-operator/issues)
-  * [KFServing](https://github.com/kubeflow/kfserving/issues)
+  * [Trainer](https://github.com/kubeflow/training-operator/issues)
+  * [Notebooks](https://github.com/kubeflow/notebooks/issues)
+  * [Dashboard](https://github.com/kubeflow/dashboard/issues)
+  * [Kserve](https://github.com/kserve/kserve/issues)
+  * [Platform / Manifests](https://github.com/kubeflow/manifests/issues)
   * [Examples](https://github.com/kubeflow/examples/issues)
```

Comment:
Let's remove this directory from here since it is out of date and not maintained currently.